tests/06-lb.sh

```diff
@@ -60,7 +62,7 @@ if [ -n "$(cilium service list)" ]; then
 fi
 
 # Add a service with ID 1
-sudo cilium service update --frontend [::]:80 --backend [::1]:90 --backend [::2]:91 --id 1 --rev 2> /dev/null || {
+sudo cilium service update --frontend [::]:80 --backends [::1]:90,[::2]:91 --id 1 --rev 2> /dev/null || {
 	abort "Service should have been added"
 }
 
```

Comment:
You do not need to change the backend to backends since I fixed it in my lb_wrr review. I also took care of the remaining non working bits in this script. I am fine either ways. If you want to commit this one I will simply ignore this file. I will still need to call it backend though.